src/archanaNJun20/Bankingsystem.java

```diff
@@ -0,0 +1,66 @@
+package archanaNJun20;
+
+public class Bankingsystem {
+
+	int balance=1000;
```

Comment:
No Need to initialize this variable. perform credit() method first.

Reply:
Done